lesson15/main.js

```diff
@@ -225,14 +204,19 @@ function validInput() {
   fetchButton.disabled = false;
 }
 
+nameBox.addEventListener("keyup", validateInputData(nameBox,namePattern,"名前を入力ください"));

```

Comment:
> 定数valueに値が格納されません。ご助言いただけないでしょうか？
やったこと
207行目の引数に値(nameBox.value)を格納してみましたが駄目でした。
ググっても適当な解決策が出てきませんでした。


以下のようにするとvalueに格納されると思います
```suggestion
nameBox.addEventListener("keyup", () => {
  validateInputData(nameBox,namePattern,"名前を入力ください")
});

```